Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -1,5 +1,24 @@
 // ArrayExtensions.swift - Copyright 2020 SwifterSwift
 
+// MARK: - Initializers
+
+public extension Array {
+    /// SwifterSwift: Creates an array with specified number of elements, for each element it calls specified closure.
+    /// - Parameters:
+    ///   - count: The number of elements in the new array.
+    ///   - element: A closure that initializes each element.
+    ///     - Parameter *index*: An index of initialized element in the array.
+    ///     - Returns: element of the array.
+    init(count: Int, element: (Int) -> Element) {
```

Comment:
Sorry just one additional comment. Could this closure be throwing and function rethrows?